DBlock Game/DBlock Game/Program.cs

```diff
@@ -1347,6 +1347,106 @@ public static void Fight()
             Menu();
         }
 
+        public static void Death()
+        {
+            Random rand = new Random();
+            string player;
+            string comp;
+            int pscore = 0;
+            int cscore = 0;
+
+            for (int i = 0; i < 4; i++)
+            {
+                player = "";
+                comp = "";
+
+                while (player != "PAPER" && player != "SCISSORS" && player != "ROCK")
+                {
+                    TextBox("you have to play paper,scisors,rock to win its a best out of 3.", Player, false);
+                    TextBox("Press 1 for paper, Press 2 for scissors, Press 3 for rock.", Player, false);
```

Comment:
this asks the player for a number between 1-3, but the following code uses uppercase strings, maybe just use numbers since they're a bit easier to work with?